### PR TITLE
Added ANON (Anonymous Bitcoin) Testnet coin config, and pool_config examples

### PR DESCRIPTION
**Pull request by 0x6b656e**

### CODE REVIEW

File: pool_configs/examples/anon.testnet.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    "address": "",
+    "_comment_address": "a transparent address to send coinbase rewards to and to transfer to zAddress.",
+
+    "zAddress": "",
+    "_comment_zAddress": "a private address used to send coins to tAddress.",
+
+    "tAddress": "",
+    "_comment_tAddress": "transparent address used to send payments, make this a different address, otherwise payments will not send",
+
+    "invalidAddress":"tAMnQVtZ3ZLZKrtcbm7Ucj2m4xnccfBo9XT",
```
Comment (Gumbees) [inline pool_configs/examples/anon.testnet.json]: Remove wallet address :).

File: pool_configs/examples/anon.testnet.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    "_comment_invalidAddress": "Invalid addresses will be converted to the above",
+
+    "walletInterval": 2.5,
+    "_comment_walletInterval": "Used to cache coin stats",
+
+    "rewardRecipients": {
+        "tANCGutikyaospw5o25UScpZ3hUEkjNisCv": 0.2
```
Comment (Gumbees) [inline pool_configs/examples/anon.testnet.json]: Same here.

File: pool_configs/examples/anon.testnet.json [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        "enabled": false,
+        "serverKey": "",
+        "serverCert": "",
+        "ca": ""
+    },
+
+    "paymentProcessing": {
+        "enabled": true,
+        "minConf": 10,
+        "paymentMode": "pplnt",
```
Comment (Gumbees) [inline pool_configs/examples/anon.testnet.json]: Change to prop. We default to prop.

File: pool_configs/examples/anon.testnet.json [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+        "_comment_paymentMode": "prop, pplnt",
+        "paymentInterval": 120,
+        "minimumPayment": 0.1,
+        "maxBlocksPerPayment": 3,
+        "daemon": {
+            "host": "127.0.0.1",
+            "port": 3127,
+            "user": "RPCUSER",
+            "password": "RPCPASSWORD"
```
Comment (Gumbees) [inline pool_configs/examples/anon.testnet.json]: Lower case please.

File: pool_configs/examples/anon.testnet.json [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    "poolId": "main",
+    "_comment_poolId": "use it for region identification: eu, us, asia or keep default if you have one stratum instance for one coin",
+
+    "daemons": [
+        {
+            "host": "127.0.0.1",
+            "port": 3127,
+            "user": "RPCUSER",
+            "password": "RPCPASSWORD"
+        }
```
Comment (Gumbees) [inline pool_configs/examples/anon.testnet.json]: lower case.

File: pool_configs/examples/anon.testnet.json [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+        "disableTransactions": true
+    },
+
+    "mposMode": {
+        "enabled": false,
+        "host": "127.0.0.1",
+        "port": 3306,
+        "user": "me",
+        "password": "mypass",
+        "database": "btcp_testnet",
```
Comment (Gumbees) [inline pool_configs/examples/anon.testnet.json]: change to anon_testnet

**Review by Gumbees — CHANGES_REQUESTED** [review]
Most of the changes are just standard, nit pickey stuff we're trying to stick to. Thanks!

**Review by 0x6b656e — COMMENTED** [review]
all changes made

**Comment by 0x6b656e** [comment]
ty for the feedback @nettts all changes made

File: pool_configs/examples/anon_testnet.json [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+        "disableTransactions": true
+    },
+
+    "mposMode": {
+        "enabled": false,
+        "host": "127.0.0.1",
+        "port": 3306,
+        "user": "me",
+        "password": "mypass",
+        "database": "btcp_testnet",
```
Comment (tarrenj) [inline pool_configs/examples/anon_testnet.json]: NIT: Please change to `anon_testnet`: https://github.com/s-nomp/s-nomp/pull/46#pullrequestreview-145656242

File: pool_configs/examples/anon_testnet.json [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+        "serverKey": "",
+        "serverCert": "",
+        "ca": ""
+    },
+
+    "paymentProcessing": {
+        "enabled": true,
+        "minConf": 10,
+        "paymentMode": "prop",
+        "_comment_paymentMode": "prop, pplnt",
```
Comment (tarrenj) [inline pool_configs/examples/anon_testnet.json:35]: @nettts I'm unsure if `pplnt` is supported: https://github.com/s-nomp/s-nomp/pull/46#pullrequestreview-145655845

**Review by tarrenj — CHANGES_REQUESTED** [review]

File: pool_configs/examples/anon_testnet.json (hunk 8 above)
Comment (0x6b656e) [reply pool_configs/examples/anon_testnet.json:35]: pplnt works, but changed it back to prop as a default

**Review by 0x6b656e — COMMENTED** [review]
last fix made

**Review by tarrenj — APPROVED** [review]

**Comment by tarrenj** [comment]
@nettts Confirmed that the changes you've requested in your review have been met.

**Review by Gumbees — APPROVED** [review]

**Comment by Gumbees** [comment]
Merged.

**Pull request merged**